src/newViews/Tagging/Sources/BulkUpload/FilesUploaded/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback, useState, useMemo } from 'react';
+import {
+    _cs,
+} from '@togglecorp/fujs';
+
+import {
+    Container,
+    List,
+} from '@the-deep/deep-ui';
+import _ts from '#ts';
+
+import FileItem from './FileItem';
+import { FileUploadResponse } from '../types';
+import FilesSearchInput from './FilesSearchInput';
+import styles from './styles.scss';
+
+const keySelector = (d: FileUploadResponse): number => d.id;
+
+interface Props {
+    className?: string;
+    onDeleteFile: (id: number) => void;
+    files: FileUploadResponse[];
+}
+
+function FilesUploaded(props: Props) {
+    const {
+        className,
+        onDeleteFile,
+        files = [],
+    } = props;
+
+    const [searchResults, setSearchResults] = useState<number[] | undefined>();
+    const fileRendererParams = useCallback((
+        _: number,
+        data: FileUploadResponse,
+    ) => ({
+        data,
+        onDeleteFile,
+    }), [onDeleteFile]);
+
+    const handleSearchFiles = useCallback((matchingFiles: number[] | undefined) => {
+        setSearchResults(matchingFiles);
+    }, []);
```

Comment:
Do we need to create a function? Why not just alias

const handleSearchFiles = setSearchResults;